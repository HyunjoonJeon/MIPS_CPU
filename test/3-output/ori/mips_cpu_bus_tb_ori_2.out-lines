register_v0:f38ffe6a
