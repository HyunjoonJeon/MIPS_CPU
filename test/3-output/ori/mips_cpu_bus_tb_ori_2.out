f38ffe6a
